.github/CODEOWNERS

```
@@ -0,0 +1 @@
+.github/workflows @toptal/site-acquisition-eng
```

```suggestion
* @toptal/site-acquisition-eng 
```

We can assign ownership for the entire repository.

---

Sure, I am fine with that